tests/test_xlson.py

```diff
@@ -122,6 +122,30 @@ def test_choose_image_field(self) -> None:
             },
         )
 
+    def test_required_field(self) -> None:
+        """Test xlson.build_field() - returns a native form choose_image field dict"""
+        options = {
+            "name": "first_name",
+            "label": "What's your name?",
+            "type": "text",
+            "required": "true",
+            "required_message": "Please enter the first name.",
+        }
```

Comment:
Could you please confirm the XLSForm JSON for a required field? I was expecting:
```python
{
  'name': 'first_name',
  'label': "What's your first name?",
  'type': 'text',
  'bind': {
    'required': 'yes',
    'jr:requiredMsg': 'Is required'
  }
}
```

Reply:
Let me look into this.

Reply:
Modified the XLSForm JSON provided to the appropriate representation.